src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,55 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int OPERATION_TYPE_POSITION = 0;
+    private static final int AMOUNT_POSITION = 1;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String reportData = createReportFromFile(fromFileName);
```

Comment:
creating report and reading from file are different operations. Split method to two ones.
```suggestion
        String dataFromFile = readDataFromFile(fromFileName);
        String report = createReport(dataFromFile);
```